tokio-test/src/io.rs

```diff
@@ -143,6 +148,81 @@ impl Builder {
     }
 }
 
+
+impl StreamBuilder  {
+    /// Return a new, empty `Builder.
+    pub fn new() -> Self {
+        Self::default()
+    }
+
+    /// call next value ?
+    pub fn poll_next(&mut self, buf: &[u8]) {
+        /// TODO
+    }
```

Comment:
You should not be able to call `poll_next` or `next` on the builder. You have to actually build it first. The io builder also doesn't implement `AsyncRead` or `AsyncWrite`.